### ISSUE TITLE
Wan 2.1 1.3B Video to video is not working

### ISSUE THREAD

**Issue by nitinmukesh** [modelscope/DiffSynth-Studio#368]
I generated a video using T2V and it produced good output.
Now I want to add some effects using V2V but unfortunately it just doesn't make any changes in video and looks same as input video. Tried different settings (denoise, seed...) as well.

```python
import torch
from diffsynth.models.model_manager import ModelManager
from diffsynth.pipelines.wan_video import WanVideoPipeline
from diffsynth.data.video import save_video, VideoData
from modelscope import snapshot_download

# Download models
snapshot_download("Wan-AI/Wan2.1-T2V-1.3B", local_dir="models/Wan-AI/Wan2.1-T2V-1.3B")

# Load models
model_manager = ModelManager(device="cpu")
model_manager.load_models(
    [
        "models/Wan-AI/Wan2.1-T2V-1.3B/diffusion_pytorch_model.safetensors",
        "models/Wan-AI/Wan2.1-T2V-1.3B/models_t5_umt5-xxl-enc-bf16.pth",
        "models/Wan-AI/Wan2.1-T2V-1.3B/Wan2.1_VAE.pth",
    ],
    torch_dtype=torch.bfloat16, # You can set `torch_dtype=torch.float8_e4m3fn` to enable FP8 quantization.
)
pipe = WanVideoPipeline.from_model_manager(
    model_manager, 
    torch_dtype=torch.bfloat16, 
    device="cuda"
)
pipe.enable_vram_management(
    num_persistent_param_in_dit=None
)
video = VideoData("video.mp4", height=480, width=832)
video = pipe(
    prompt="Cinematic fantasy sequence, an elderly wizard with a flowing white beard and ornate blue robes gallops across a misty meadow on a powerful white stallion. The wizard holds an intricately carved wooden staff high in his right hand, now crackling with magical energy. A radiant blue glow pulses from the staff's engravings, emitting flickering tendrils of mystical lightning. Wisps of golden and cyan energy swirl around the staff, leaving behind faint shimmering trails as the horse charges forward. His robes billow dramatically, and the early morning light filters through the ancient oak trees, casting golden highlights across the magical glow. The distant mountains and a stone castle perched on a hill add depth to the medieval fantasy setting. The cinematic tracking shot captures the wizard and his enchanted staff in smooth, fluid motion.",
    negative_prompt="Static pose, blurry details, text overlays, cartoon style, illustrated look, anime, cell-shaded, paintings, frozen frame, poor quality, low resolution, compression artifacts, distorted proportions, anatomical errors, extra limbs, poorly rendered features, unnatural movements, busy background, modern elements, vehicles, phones, unrealistic horse anatomy, floating rider, disconnected rider, multiple horses, multiple riders, unnatural lighting effects, overexposed glow, harsh neon colors, unnatural VFX, artificial CGI look, plastic textures.",
    input_video=video, 
    denoising_strength=0.7,
    num_inference_steps=50,
    seed=12345567, 
    tiled=True
)
save_video(video, "video2.mp4", fps=15, quality=10)

```

Video
https://drive.google.com/file/d/1y5JyM2ja79JbUO5AL0HUk3eG4PWxmlvW/view?usp=sharing

**Comment by FurkanGozukara** [comment]
it works but your video must match frame count

i made that in my app and auto crops : https://youtu.be/hnAhveNy-8s

**Comment by Artiprocher** [comment]
@nitinmukesh Hi! I think the `num_frames` doen't match your input video. This model requires that `num_frames % 4 == 1`. You can use the following code to cut the input_video.
```python
video = VideoData("video.mp4", height=480, width=832)
video = [video[i] for i in range(81)]
```

**Comment by nitinmukesh** [comment]
@Artiprocher 

Thank you for updated code. Unfortunately same old result, the video is not getting updated. I am using the video generated from T2V and using it in V2V.

**Comment by nitinmukesh** [comment]
Just to ensure that the prompt is correct I used the prompt with T2V and it worked
https://drive.google.com/file/d/1cf5SPCiNOLa5Nj_VeGoXRavDMQeG2oaY/view?usp=sharing

So not sure what is wrong with V2V. Anything else that I can try

**Comment by nitinmukesh** [comment]
> i made that in my app and auto crops 
Auto crop is already included in VideoData. 
If you look at the code, crop_and_resize is used

```python
    def __getitem__(self, item):
        frame = self.data.__getitem__(item)
        width, height = frame.size
        if self.height is not None and self.width is not None:
            if self.height != height or self.width != width:
                frame = crop_and_resize(frame, self.height, self.width)
        return frame
```

**Comment by Artiprocher** [comment]
@nitinmukesh I cannot open your video file using QuickTime. Maybe something is wrong in this file.

**Comment by nitinmukesh** [comment]
It is working now. Thank you.
